src/bindings/python/flux/uri/uri.py

```diff
@@ -8,6 +8,7 @@
 # SPDX-License-Identifier: LGPL-3.0
```

Comment:
commit message typo "fi" -> "if"

`Then, in the JobURI str() method, return the local URI fi force_local is True.`